ssl/ssl_lib.c

```diff
@@ -3736,14 +3734,24 @@ SSL_CTX *SSL_CTX_new_ex(OSSL_LIB_CTX *libctx, const char *propq,
     /* initialize cipher/digest methods table */
     if (!ssl_load_ciphers(ret))
         goto err;
+
+    if (!ssl_load_groups(ret))
+        goto err;
+
+    /* load provider sigalgs */
+    if (!ssl_load_sigalgs(ret))
+        goto err;
+
     /* initialise sig algs */
     if (!ssl_setup_sig_algs(ret))
         goto err;
 
-    if (!ssl_load_groups(ret))
+    if (!SSL_CTX_set_ciphersuites(ret, OSSL_default_ciphersuites())) {
+        ERR_raise(ERR_LIB_SSL, ERR_R_SSL_LIB);
         goto err;
+    }
 
-    if (!SSL_CTX_set_ciphersuites(ret, OSSL_default_ciphersuites())) {
+    if ((ret->cert = ssl_cert_new(SSL_PKEY_NUM+ret->sigalg_list_len)) == NULL) {
```

Comment:
Nit: spaces around`+`

Reply:
So changed in 0ac814f8247d1a5f157eec2b62bce0000c3a2d1e.